index.js

```diff
@@ -183,6 +185,53 @@ class GithubScm extends Scm {
         };
     }
 
+    /**
+     * Promise to wait a certain number of seconds
+     *
+     * Might make this centralized for other tests to leverage
+     *
+     * @method promiseToWait
+     * @param  {Number}      timeToWait  Number of seconds to wait before continuing the chain
+     * @return {Promise}
+     */
+    promiseToWait(timeToWait) {
+        return new Promise((resolve) => {
+            setTimeout(() => resolve(), timeToWait * 1000);
+        });
+    }
+
+    /**
+     * Wait computing mergeability
+     * @async  getPrMergeable
+     * @param  {Object}   config
+     * @param  {String}   config.scmUri     The scmUri to get PR info of
+     * @param  {String}   config.token      The token used to authenticate to the SCM
+     * @param  {Integer}  config.prNum      The PR number used to fetch the PR
+     * @param  {Integer}  count             The polling count
+     * @return {Promise}                    Resolves to object containing result of computing mergeability
+     *                                      The parameter of success exists for testing
+     */
+    async waitPrMergeability({ scmUri, token, prNum }, count) {
+        if (count >= POLLING_MAX_ATTEMPT) {
+            winston.warn('Computing mergerbility did not finish');
+
+            return { success: false };
+        }
+        try {
+            const { mergeable } = await this.getPrInfo({ scmUri, token, prNum });
+
+            if (mergeable !== null) {
```

Comment:
I think we should add to a comment why we compare `mergeable` to `null`, not undefined or just `if (mergeable)`. 

> https://developer.github.com/v3/pulls/
> If the value is null, then GitHub has started a background job to compute the mergeability.

Reply:
Thanks. I fixed it.